src/snapred/backend/data/LocalDataService.py

```diff
@@ -460,7 +460,7 @@ def writeNormalizationWorkspaces(self, record: NormalizationRecord):
         for workspace in record.workspaceNames:
             ws = mtd[workspace]
             if ws.isRaggedWorkspace():
-                filename = Path(workspace + ".tar")
+                filename = Path(workspace + ".nxs.h5")
```

Comment:
This single line is the cause for the codecov complaint.  I don't think it's worth worrying over.